defaults.lua

```diff
@@ -198,4 +198,7 @@ DUSE_TURBO_LIB = false,
 -- Absolute path to stardict files (override)
 -- By default they're stored in data/dict under dataDir.
 STARDICT_DATA_DIR = nil,
+
+-- Rotate images clockwise (90) counter-clockwsie (-90) or not at all (0) to match their aspect ratio with the screen
+IMAGE_FIT_ROTATE = 0,
```

Comment:
We've mostly been trying to move things out of here to the GUI, but additionally this isn't even used? ;-)

Reply:
I might have misunderstood what `defaults.lua` does.

Reply:
Where you say `local image_fit_rotate = tonumber(G_reader_settings:readSetting("image_fit_rotate", 0))` that would have to be IMAGE_FIT_ROTATE instead of 0 to use it. But please just delete these lines instead.